azext_iot/digitaltwins/commands_identity.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.digitaltwins.providers.resource import ResourceProvider
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def assign_identity(
+    cmd,
+    name,
+    system_identity=None,
```

Comment:
I encourage type annotations for new code

Reply:
will add